docker-entrypoint.sh

```diff
@@ -35,12 +35,27 @@ drop_privs_cmd() {
     fi
 }
 
+copy_plugins_if_required() {
+  echo "Enabling required built-in plugins"
+  for TARGET_PLUGIN in $(echo "$ENABLE_BUILT_IN_PLUGINS" | grep -o -e "[^;]*"); do
+    echo "Moving $TARGET_PLUGIN to plugin directory"
```

Comment:
```suggestion
    echo "Moving ${TARGET_PLUGIN} to plugin directory"
```

(style convention of this file)

Reply:
(also in `echo` strings below)